examples/aot_autotuning_example.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+"""
+AOT Autotuning Example
+======================
+
+This example demonstrates how to use the AOT (Ahead-of-Time) autotuning
+workflow for Helion kernels.
+
+The AOT workflow consists of three phases:
+1. Collect: Run benchmarks, autotuning each shape individually
+2. Measure: Re-run benchmarks, measuring all configs across all shapes
```

Comment:
(as we discussed, maybe "all shapes" is not exactly accurate, as user can customize what shapes to run in each phase)